src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot.subsystems.conveyor;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+/**
+ * Add new functions- motor, stopper, funnelProximity, shooterProximity, ballNum, newFunnelBall, newShooterBall.
+ */
+public class Conveyor extends SubsystemBase {
+    private final TalonSRX motor = new TalonSRX(Ports.Conveyor.MOTOR_POWER);
+    private final Solenoid stopper = new Solenoid(Ports.Conveyor.SOLENOID);
+    private final AnalogInput funnelProximity = new AnalogInput(Ports.Conveyor.SHOOTER_PROXIMITY);
+    private final AnalogInput shooterProximity = new AnalogInput(Ports.Conveyor.FUNNEL_PROXIMITY);
+    private int ballNum = Constants.Conveyor.BALL_NUMBER;
```

Comment:
You can give these fields numeric strating values. You don't need the constants.
private int ballNum = 3;

Reply:
Nope